projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -1,43 +1,62 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 use Automattic\Jetpack\Connection\Initial_State as Connection_Initial_State;
 use Automattic\Jetpack\Connection\Manager as Connection_Manager;
 use Automattic\Jetpack\Status;
 
 require_once __DIR__ . '/class.jetpack-admin-page.php';
 require_once __DIR__ . '/class-jetpack-redux-state-helper.php';
 
-// Builds the landing page and its menu
+/**
+ * Builds the landing page and its menu.
+ */
 class Jetpack_React_Page extends Jetpack_Admin_Page {
-
+	/**
+	 * Show the landing page only when Jetpack is connected.
+	 *
+	 * @var boolean
```

Comment:
It seems we normally use "bool" rather than "boolean" in PHP.
```suggestion
	 * @var bool
```